.NET/Microsoft.Recognizers.Text.DateTime/Utilities/TasksModeProcessing.cs

```diff
@@ -84,7 +222,14 @@ private static DateTimeParseResult TasksModeModifyDateValue(DateTimeParseResult
                         var inputDay = inputTime.Day;
                         var inputMonth = inputTime.Month;
 
-                        if (inputDay == referenceTime.Day && inputMonth == referenceTime.Month)
+                        if (slot.Text.Contains("next week") && !slot.TimexStr.Contains("XXXX"))
```

Comment:
Please don't hardcode language terms like "next week". These should ideally reuse resources or last case be a constant.
Also, the same for timex pieces like XXXX and others.

Reply:
timex pieces are not language specific, so I have used them as it is in file.
Will replace "next week" term with regex or constant.

Reply:
replace hardcoded string with constants